source/common/ssl/context_impl.cc

```diff
@@ -17,42 +17,6 @@
 namespace Envoy {
 namespace Ssl {
 
-/**
- * The following function was generated with 'openssl dhparam -C 2048'
- */
-DH* get_dh2048() {
```

Comment:
Can we get some explanation of why this code and the code below has been deleted in the commit message?